src/components/pill-selector-dropdown/Pill.js

```diff
@@ -3,19 +3,29 @@ import React from 'react';
 import classNames from 'classnames';
 
 type Props = {
+    isDisabled?: boolean,
     isSelected?: boolean,
     isValid?: boolean,
     onRemove: Function,
     text: string,
 };
 
-const Pill = ({ isSelected = false, isValid = true, onRemove, text }: Props) => (
-    <span className={classNames('pill', { 'is-selected': isSelected, 'is-invalid': !isValid })}>
-        <span className="pill-text">{text}</span>
-        <span aria-hidden="true" className="close-btn" onClick={onRemove}>
-            ✕
+const Pill = ({ isDisabled = false, isSelected = false, isValid = true, onRemove, text }: Props) => {
+    const removeFn = isDisabled ? undefined : onRemove;
+    return (
+        <span
+            className={classNames('pill', {
+                'is-selected': isSelected,
+                'is-invalid': !isValid,
+                'is-disabled': isDisabled,
+            })}
+        >
+            <span className="pill-text">{text}</span>
+            <span aria-hidden="true" className="close-btn" onClick={removeFn}>
+                ✕
```

Comment:
Wondering if we should use the proper [HTML Entity](https://dev.w3.org/html5/html-author/charref) here to avoid encoding issues.

`{'&cross;'}` or however you do it in react?

Reply:
Hmm, `&cross;` doesn't look very good, and neither does `&times;`. In our modals we use an icon, but I'm not sure if that's what we want here too?

This is with `&cross;`:
![Screen Shot 2019-07-02 at 10 42 33 AM](https://user-images.githubusercontent.com/1280912/60534411-a227a980-9cb6-11e9-8654-fa8154e43f0c.png)

Reply:
yeah I would leave it at X for now